src/pages/waiting-room-to-car/waiting-room-to-car.scss

```diff
@@ -1,12 +1,37 @@
 page-waiting-room-to-car {
   ion-content > div {
     background-color: map-get($colors, "mes-white");
-    .mes-full-width-card {
+    .mes-validated-row {
+      position: relative;
+      min-height: 120px;
       border-bottom: 2px solid map-get($colors-gds, 'gds-grey-5');
+      .spacing-row {
+        height: 32px;
+      }
+      .validation-message-row {
+        height:32px;
+      }
+      .component-label {
+        padding: 24px;
+      }
+      .row-validation-bar {
+        position: absolute;
+        width: 8px;
+        height: 100%;
+        &.ng-invalid {
+          background-color: map-get($colors, mes-validation-error);
+        }
+      }
+    }
+    .mes-full-width-card {
+      height: 120px;
       #continue-to-test-report {
         width: 100%;
         height: 56px;
       }
     }
+    .no-padded-col {
```

Comment:
I think Ionic gives a `no-padding` directive out of the box that we use in quite a few other places.

Reply:
we did try no-padding originally, and it didn't seem to work but this may have been due to other reasons. used no-padding directive and looks good.